tests/unit_tests/test_model_triso.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+
+from math import pi
+
+import numpy as np
+from numpy.linalg import norm
+import openmc
+import openmc.model
+import pytest
+import scipy.spatial
+
+
+_PACKING_FRACTION = 0.35
+_RADIUS = 4.25e-2
+
+
+@pytest.fixture(scope='module',
+                params=[{'shape': 'cube',
+                         'length': 0.75,
+                         'radius': 0.,
+                         'volume': 0.75**3},
+                        {'shape': 'cylinder',
+                         'length': 0.5,
+                         'radius': 0.5,
+                         'volume': 0.5*pi*0.5**2},
+                        {'shape': 'sphere',
+                         'length': 0.,
+                         'radius': 0.5,
+                         'volume': 4/3*pi*0.5**3}])
```

Comment:
Seems a little awkward to have a decorator that spans so many lines. Maybe change to something like:
```Python
domain_params = [
    {'shape': 'cube', 'length': 0.75, 'radius': 0., 'volume': 0.75**3},
    {'shape': 'cylinder', 'length': 0.5, 'radius': 0.5, 'volume': 0.5*pi*0.5**2},
    {'shape': 'sphere', 'length': 0., 'radius': 0.5, 'volume': 4/3*pi*0.5**3}
]


@pytest.fixture(scope='module', params=domain_params,
                ids=['cube', 'cylinder', 'sphere'])
```